src/test/java/com/mediamarktsaturn/technolinator/sbom/CdxgenClientEnvTest.java

```diff
@@ -170,7 +170,7 @@ void testWithoutArgs() {
             .doesNotContainKey("GRADLE_ARGS")
             .containsEntry("CDXGEN_TIMEOUT_MS", Long.toString(60 * 60 * 1000))
             .containsEntry("MVN_ARGS", "-B -ntp")
-            .containsEntry("CDX_MAVEN_INCLUDE_TEST_SCOPE", "false")
+            .containsEntry("CDX_MAVEN_INCLUDE_TEST_SCOPE", "true")
```

Comment:
hasn't this been removed?

Reply:
we derive this setting now from `REQUIRED_SCOPE_ONLY` - what's `false` by default, that means from now on, test scope on maven projects is included - like for all other ecosystems ;)